Create suzumi.badboy.json

s3mant opened the pull request.

s3mant committed:
Create api.badboy.json

MaskDuck commented on api.badboy.json:
```diff
@@ -0,0 +1,11 @@
+  "description": "Bad Boy's Api",
+  "repo": "https://github.com/Bad-Boy-Codes/bad-boy-codes.github.io",
+  "owner": {
+    "username": "badboy",
```
username is wrong ig
```suggestion
    "username": "Bad-Boy-Codes",
```

MaskDuck commented:
and also it must be placed in the domains folder

Bad-Boy-Codes commented:
> and also it must be placed in the domains folder

oh lol okay xd

MaskDuck commented:
idk

s3mant committed:
Update and rename api.badboy.json to domains/api.badboy.json

s3mant committed:
did nothing?

s3mant reviewed (commented):
a

MaskDuck reviewed (dismissed):
not sus

s3mant committed:
okay i changed my mind

i don't need extra domain for my api it's in the main domain, and now using this for my discord bot's dashboard
